jujugui/static/gui/src/app/views/utils.js

```diff
@@ -621,8 +621,18 @@ YUI.add('juju-view-utils', function(Y) {
     @param {Object} db Reference to the app db.
   */
   utils.exportEnvironmentFile = function(db, legacyServicesKey) {
-    var result = db.exportDeployer(legacyServicesKey);
+    const apps = db.services.toArray();
+    const idMap = new Map();
+    // Store a map of all the temporary app ids to the real ids.
+    apps.forEach(app => {
+      idMap.set(app.get('id'), app.get('name'));
+    });
+    var result = db.exportDeployer(legacyServicesKey, true);
```

Comment:
That's an interesting idea, I did this using the hardcoded value so that we can change it in the future without having to add the code back in. I guess we could do a followup to add the confirm if you think it'd be nice?